src/libopensc/card-cardos.c

```diff
@@ -1003,8 +1224,7 @@ cardos_decipher(struct sc_card *card,
 	card->max_send_size = card_max_send_size;
 	card->reader->max_send_size = reader_max_send_size;
 
-	return r;
-}
+	SC_FUNC_RETURN(card->ctx, SC_LOG_DEBUG_VERBOSE, r);}
```

Comment:
can you fix the formatting here?

Reply:
OK, fixed in my source. Before final release, I need to remove the DEE_TEST stuff too.